package.json

```diff
@@ -20,7 +20,7 @@
     "start": "NODE_ENV=production node server.js",
     "build": "./node_modules/.bin/webpack",
     "dev": "NODE_ENV=development supervisor --ignore ./node_modules/,./public/js server.js",
-    "test": "export PORT=4243 ; jest && jshint server.js lib test && jsxhint public/js/**/*.jsx && mocha test test/*.js && npm run coverage",
+    "test": "export PORT=4243 ; jshint server.js lib test && jscs server.js lib test && jsxhint public/js/**/*.jsx && jest && mocha test test/*.js && npm run coverage",
```

Comment:
The test script is getting long. Maybe it'd be nicer to have it as a shell script in the `scripts/` folder?